app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -36,6 +36,7 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.concurrent.Future.successful
 
+//noinspection ScalaStyle
```

Comment:
What was this for?

Reply:
Ahh, stops intellij moaning there's more than 30 methods in the class, and removes the underlined squiggle from everything so it's readable. Meant to remove that before committing.

Reply:
I've removed it now